src/api/app/models/bs_request.rb

```diff
@@ -730,7 +730,7 @@ def change_review_state(new_review_state, opts = {})
 
       # we want to check right now if pre-approved requests can be processed
       if go_new_state == :new && accept_at
-        Delayed::Job.enqueue AcceptRequestsJob.new
+        AcceptRequestsJob.perform_later
```

Comment:
I don't get what you mean. You want me to test that the job gets scheduled by `.change_review_state`?